repondeur/zam_repondeur/templates/article_edit.html

```diff
@@ -44,7 +36,7 @@ <h2>{{ amendement.subdiv_disp }}</h2>
         </div>
         <div class="form-group">
             <input type="submit" name="submit" value="Enregistrer" class="btn btn-primary">
-            <a href="{{ request.route_url('list_amendements', chambre=amendement.chambre, session=amendement.session, num_texte=amendement.num_texte, organe=amendement.organe) }}" class="btn">Annuler</a>
+            <a href="{{ request.resource_url(context.__parent__.__parent__['amendements']) }}" class="btn">Annuler</a>
```

Comment:
Could be `context.parent.parent['amendements']` if I understand the latest changes correctly?

Reply:
Thought I converted them all but looks like some have slipped through!